actionpack/test/dispatch/debug_locks_test.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+
+class DebugLocksTest < ActionDispatch::IntegrationTest
+  def setup
+    build_app
```

Comment:
Needs to call `super` or use `setup` with a block.

Reply:
Thanks! I missed. I updated to use `setup` with a block.